specification/resourcegraph/resource-manager/readme.md

```diff
@@ -48,6 +57,7 @@ These settings apply only when `--tag=package-preview-2020-04` is specified on t
 ``` yaml $(tag) == 'package-preview-2020-04'
 input-file:
   - Microsoft.ResourceGraph/preview/2020-04-01-preview/resourcegraph.json
+  - Microsoft.ResourceGraph/preview/2020-04-01-preview/resourcechanges.json
```

Comment:
Let's add    Microsoft.ResourceGraph/preview/2018-09-01-preview/graphquery.json in 2020-04 as well as 2020-09 packages as this was missed before also.